### ISSUE TITLE
Export Parameter values of objects & relationships

### ISSUE THREAD

**Issue by JaKehrl** [spine-tools/Spine-Toolbox#2050]
Hi community, 

is it possible to export the input parameters from objects and relationships as per attached image?
Also, would it be possible to export an overview of which parameter values are used in which scenario? 

<img width="473" alt="image" src="https://user-images.githubusercontent.com/124700754/227184965-ca5ab25e-2c7a-47dd-b4bf-b00581c49196.png">

**Comment by jkiviluo** [comment]
Yes, it's possible, but there are still some limitations. You need to map alternative to a column or a row in the exporter mappings, like here:
![image](https://user-images.githubusercontent.com/40472544/227441003-60fbb139-ea91-418c-a9c3-baecae3153d3.png)

The remaining issue is that you must not select any scenarios in the filter and then it will export all scenarios to the csv file. If you select one or more scenarios, they will be exported to separate files.
